ui/translations/en-us.yaml

```diff
@@ -150,6 +150,9 @@ form:
     variable_name_placeholder: 'var_key'
     variable_value: 'Value'
     variable_value_placeholder: 'var_value'
+    variable_set_sensitive: Set as sensitive
+    variable_value_placeholder_sensitive: 'Sensitive - write only'
+    variable_form_placeholder_sensitive: 'sensitive - write only'
```

Comment:
(suggestion) Have the casing for both be 'Sensitive - write only' so that they match?

Reply:
This is the way it is in HCP and the accompanying designs so I'm just making them match.

Reply:
Makes sense, but we should follow https://design-system-components-hashicorp.vercel.app/content/writing-guidelines in the future to be consistent with casing

Reply:
@joshklekamp or @ryenotbread what do you think? This difference in case is how it works in TFC which is what I modeled ours on.

Reply:
I think they should match and they should be sentence cased.

Reply:
I agree they should match and be sentence cased.